webapp/utils/constants.jsx

```diff
@@ -59,7 +59,8 @@ export const Preferences = {
     EMAIL_INTERVAL: 'email_interval',
     INTERVAL_IMMEDIATE: 30, // "immediate" is a 30 second interval
     INTERVAL_FIFTEEN_MINUTES: 15 * 60,
-    INTERVAL_HOUR: 60 * 60
+    INTERVAL_HOUR: 60 * 60,
+    INTERVAL_NEVER: 'never'
```

Comment:
Can you change INTERVAL_NEVER to be a number? I think 0 should work since immediate is stored as 30. Having it sometimes be a number and sometimes a string could cause problems, particularly on the server where the type system is much stronger